### PR TITLE
Update mnist_gcp.ipynb - increase training step value

### PR DESCRIPTION
**Pull request by amygdala**
Increasing the train_steps value allows training progress info to show up in the logs.  (Apparently 200 steps was too small for that).

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/examples/747)

### CODE REVIEW

**Comment by amygdala** [comment]
It doesn't seem that the papermill-related presubmit error is related to my mini change.

**Comment by jlewi** [comment]
https://github.com/kubeflow/examples/issues/750 is tracking the notebook test failures.

**Comment by jlewi** [comment]
@amygdala how much longer does training take if we increase from 200 to 2000 steps?

I'd like the notebook to run e2e pretty quickly say <=10 minutes. This is useful for testing (at least until we parameterize it better).

But also I think its valuable if people can just do "Run all" and have the notebook complete fairly quickly even if that means training a poor quality model.

Could we add a note instead along the lines of "To train a high quality model increase the number of steps to X? and it will take this long?"

Perhaps we could produce a graph from tensorboard that shows accuracy and training time as a function of steps and then include that in the markdown so that we can provide good guidance on the tradeoff between steps, time, and accuracy?

**Comment by amygdala** [comment]
It actually didn't seem to take appreciably longer (b/c the model is so simple, it trains very quickly) but I will leave it up to you.  If you'd rather leave it like it is, that's fine.

**Comment by jlewi** [comment]
@amygdala Did you want to rebase this and fix or should we just close this PR?

**Comment by amygdala** [comment]
let's just close it, I don't think it's that important.
